hack/run-e2e-test-operator.sh

```diff
@@ -1,10 +1,8 @@
 #!/bin/bash -xe
 
-${KUBECTL} apply -f ${MANIFESTS_DIR}/namespace.yaml
-
 mkdir -p test_logs/e2e/operator
 unset GOFLAGS && ${OPERATOR_SDK} test local ./test/e2e/operator \
 	--kubeconfig ${KUBECONFIG} \
 	--namespace ${OPERATOR_NAMESPACE} \
-	--up-local \
+	--no-setup \
```

Comment:
Yep, --up-local was stoopid from my part :-/

Reply:
Nah. It worked to get it up and running to begin with.